src/sgame/sg_bot_local.h

```diff
@@ -87,6 +89,40 @@ struct botSkill_t
 	float aimShake;
 };
 
+// boolean flags that tells which skill (compétence) the bot has.
+//
+// When you add a skill, add it to the skill tree in sg_bot_skilltree.cpp and
+// change the number of skillpoint a bot has in BotDetermineSkills
+enum bot_skill
+{
+	// movement skills
+	BOT_B_BASIC_MOVEMENT, // doesn't do anything as of now
+	BOT_A_MARA_JUMP_ON_ATTACK,
+	BOT_A_LEAP_ON_ATTACK, // mantis
+	BOT_A_POUNCE_ON_ATTACK, // dragoon and adv dragoon
+	BOT_A_TYRANT_CHARGE_ON_ATTACK,
+
+	// situation awareness and survival
+	BOT_B_PAIN, // basic awareness: notice an enemy if it bites you, or shoots at you
+	BOT_A_MARA_JUMP_ON_FLEE,
+	BOT_A_LEAP_ON_FLEE, // mantis
+	BOT_A_POUNCE_ON_FLEE, // dragoon and adv dragoon
+	BOT_A_TYRANT_CHARGE_ON_FLEE,
+	BOT_H_RUN_ON_FLEE, // when fleeing, RUN
+	BOT_H_BUY_ARMOR, // knows armor exists at all, if the bot doesn't have BOT_H_PREFER_ARMOR too it will always prefer to buy guns
+	BOT_H_PREFER_ARMOR, // prefer to buy armor rather than guns
+	BOT_H_MEDKIT, // knows the medkit even exist
+
+	// fighting skills
+	BOT_B_BASIC_FIGHT, // doesn't do anything as of now
```

Comment:
Why would you have a skill that doesn't do anything?

Reply:
Because it encourages bot specialisation in some "knowledge domain"

Reply:
So there are some skills that depend on it? Which ones are they?

Reply:
Never mind I found it